includes/polling/mempools/hrstorage.inc.php

```diff
@@ -11,12 +11,25 @@
 }
 
 $entry = $storage_cache['hrstorage'][$mempool['mempool_index']];
-
 $mempool['units'] = $entry['hrStorageAllocationUnits'];
-$mempool['used'] = ($entry['hrStorageUsed'] * $mempool['units']);
+$has_avail = false;  // sysAvail based calculation when available
```

Comment:
the unit test is still complaining after tying with this flag, any ideas? @murrant

Reply:
$has_avail will not be set anywhere...
Ok, i think this is a reasonable work-around. Make a new RRD if sysAvail exists, check if the rrd exists in the graph and then add it to the graph.

Reply:
orrr, simply replace "free" in the rrd with available if it exists... let me ask around on this one.

Reply:
sure, I feel we are almost there..